### ISSUE TITLE
Auto-Generating material files not working with BasicGradleGame

### ISSUE THREAD

**Issue by yaRnMcDonuts** [jMonkeyEngine/sdk#305]
I cannot create an auto-generated material file while using the SDK's scene composer in a BasicGradleGame based project. 

Whenever I press the drop down to choose a material for a selected model, I choose the "create j3m file" option, but the material file that is created in the "generated" folder is empty.

**Comment by neph1** [comment]
It seems it's having trouble finding the MatDefs from core. That drop down menu is also empty. But if I drop a matdef into the MatDef folder of the project, I can select it.

**Comment by yaRnMcDonuts** [comment]
Yes I notice that as well. The drop down menu contains all my own shaders located in my projects assets folder, but none of the MatDefs from dependencies such as JME core or Terrain.

I am also noticing that the material editor begins to work again with my GradleTemplate based project when I open up a material using a MatDef from my assets folder, so I believe if this issue is resolved then it will also resolve the other issue with Gradle projects in the SDK where the visual Material Editor doesn't work for the stock Lighting.j3md and PBRLighting.3md MatDefs
